app/views/users/index.html.erb

```diff
@@ -1 +1,17 @@
-<h1>This page shows all users</h1>
\ No newline at end of file
+<main>
+    <div class="users-container" >
+      <% @users.each do |user| %>
+    
+      <div class="single-user">
+        <div class="user-image">
+          <img src="<%= user.photo%>" alt="profile" class="user-profile">
+        </div>
+        <div class="user-information">
+          <h2 class="user-name"><%= user.name%></h2>
+            <span class="post-counter"><p class="count">Number of posts: <%= user.posts.count %></p></span>
```

Comment:
- [ ] As per the requirement, clicking on each user must redirect us to the user show page. Kindly implement this.